tests/unit/helpers.js

```diff
@@ -1558,9 +1559,35 @@ export const screen = {
   ...customQueries,
 };
 
+const getByTextInDocument = (text) => {
```

Comment:
can you explain what this function is for, contrasting to `getByText`

Reply:
I think that's a better name, yes.  (I thought of `getByFlatText` but I'm not convinced its better 🤷 )